certified-connectors/dvelop/apiDefinition.swagger.json

```diff
@@ -835,6 +835,474 @@
+              "value-path": "value",
+              "value-collection": "data",
+              "value-title": "displayName",
+              "parameters": {
+                "defaultAction": "updatedossier",
+                "paramDef": "k1:queryType|k2:repositoryId",
+                "k1": "docTypeDossier",
+                "k2": {
+                  "parameter": "repositoryId"
+                }
+              }
+            },
+            "x-ms-dynamic-list": {
+              "operationId": "DefaultGetData",
+              "itemValuePath": "value",
+              "itemsPath": "data",
+              "itemTitlePath": "displayName",
+              "parameters": {
+                "defaultAction": {
+                  "value": "updatedossier"
+                },
+                "paramDef": {
+                  "value": "k1:queryType|k2:repositoryId"
+                },
+                "k1": {
+                  "value": "docTypeDossier"
+                },
+                "k2": {
+                  "parameterReference": "repositoryId"
+                }
+              }
+            }
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "x-ms-dynamic-schema": {
+                "operationId": "DefaultGetRequestSchema",
+                "value-path": "request",
+                "parameters": {
+                  "defaultAction": "updatedossier",
+                  "paramDef": "k1:repositoryId|k2:docTypeId",
+                  "k1": {
+                    "parameter": "repositoryId"
+                  },
+                  "k2": {
+                    "parameter": "categoryId"
+                  },
+                  "k3": "dummy",
+                  "k4": "dummy",
+                  "k5": "dummy"
+                }
+              },
+              "x-ms-dynamic-properties": {
+                "operationId": "DefaultGetRequestSchema",
+                "parameters": {
+                  "defaultAction": {
+                    "value": "updatedossier"
+                  },
+                  "paramDef": {
+                    "value": "k1:repositoryId|k2:docTypeId"
+                  },
+                  "k1": {
+                    "parameterReference": "repositoryId"
+                  },
+                  "k2": {
+                    "parameterReference": "categoryId"
+                  },
+                  "k3": {
+                    "value": "dummy"
+                  },
+                  "k4": {
+                    "value": "dummy"
+                  },
+                  "k5": {
+                    "value": "dummy"
+                  }
+                },
+                "itemValuePath": "request"
+              }
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Response in case of a successful dossier update.",
```

Comment:
@BeritHorst please provide a more precise description for 2xx codes, the part with `Response in case of ...` is not necesssary here.